extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcConfig.java

```diff
@@ -94,6 +96,28 @@
     @ConfigItem(defaultValue = "true")
     public boolean autoProducerMethods;
 
+    /**
+     * The list of types that should be excluded from discovery.
+     * <p>
+     * An element value can be:
+     * <ul>
+     * <li>a fully qualified class name, i.e. {@code org.acme.Foo}</li>
+     * <li>a simple class name as defined by {@link Class#getSimpleName()}, i.e. {@code Foo}</li>
+     * <li>a package name with suffix {@code .*}, i.e. {@code org.acme.*}, matches a package</li>
+     * <li>a package name with suffix {@code .**}, i.e. {@code org.acme.**}, matches a package that starts with the value</li>
+     * </ul>
+     * If any element value matches a discovered type then the type is excluded from discovery, i.e. no beans and observer
+     * methods are created from this type.
+     */
+    @ConfigItem
+    public Optional<List<String>> excludeTypes;
+
+    /**
+     * Artifacts on the class path that should be excluded from discovery even though the .
```

Comment:
Something seems to be missing in the comment here :)

Reply:
Good catch ;-)